source/nordic-sdk/components/softdevice/s130/headers/nrf_svc.h

```diff
@@ -49,7 +49,7 @@
   { \
     __asm( \
         "svc %0\n" \
-        "bx r14" : : "I" ((uint32_t)number) : "r0" \
```

Comment:
It seems that this file is generating way too many warning of type `-Wunused-function` and `-Wunused-parameter`. Any ideas on how to modify this code to fix the problem?